packages/esm-appointments-app/src/tabs/appointments-table.resource.ts

```diff
@@ -90,201 +26,75 @@ interface Encounter {
   voided: boolean;
 }
 
-interface Appointment {
-  diagnoses: Array<any>;
-  encounterDatetime: string;
-  encounterProviders?: Array<{ provider: { person: { display: string } } }>;
-  encounterType: { display: string; uuid: string };
-  obs: Array<ObsData>;
-  voided: boolean;
-
-  // uuid: string;
-  // appointmentNumber: string,
-  // patient: Object,
-  // service: Object,
-  // serviceType": string,
-  // provider: string,
-  // location: Object,
-  // startDateTime: Timestamp,
-  // endDateTime: Timestamp,
-  // appointmentKind: string,
-  // status: string,
-  // comments: null,
-  // additionalInfo: null,
-  // teleconsultation: null,
-  // providers: [],
-  // voided: false,
-  // extensions: Object,
-  // "teleconsultationLink": null,
-  // "recurring": false
+export interface AppointmentsFetchResponse {
+  data: Array<Appointment>;
 }
 
-interface MappedEncounter extends Omit<Encounter, 'encounterType' | 'provider'> {
-  encounterType: string;
-  provider: string;
+export interface Appointment {
+  appointmentKind: string;
+  appointmentNumber: string;
+  comments: string;
+  endDateTime: Date | number;
+  location: OpenmrsResource;
+  patient: fhir.Patient;
+  provider: OpenmrsResource;
+  providers: Array<OpenmrsResource>;
+  recurring: boolean;
+  service: AppointmentService;
+  startDateTime: number | any;
```

Comment:
```suggestion
  startDateTime: string;
```
Most of the time dates with openmrs rest are returned as string